pkg/config/piped.go

```diff
@@ -310,7 +310,25 @@ type CloudProviderCloudRunConfig struct {
 }
 
 type CloudProviderLambdaConfig struct {
+	// The region to send requests to. This parameter is required.
+	// e.g. "us-west-2"
+	// A full list of regions is: https://docs.aws.amazon.com/general/latest/gr/rande.html
 	Region string `json:"region"`
+	// The Amazon Resource Name (ARN) of the function's execution role.
+	Role string `json:"role"`
+	// Path to the shared credentials file.
+	//
+	// Piped attempts to retrieve credentials in the following order:
+	// 1. from the environment variables. Available environment variables are:
+	//   - AWS_ACCESS_KEY_ID or AWS_ACCESS_KEY
+	//   - AWS_SECRET_ACCESS_KEY or AWS_SECRET_KEY
+	// 2. from the given credentials file.
+	// 3. from the EC2 Instance Role
+	CredentialsFile string `json:"credentialsFile"`
```

Comment:
[rfc] not sure this `Role` configuration should be apart of `CloudProviderLambdaConfig` or `FunctionManifest`. In my opinion, one lambda cloud provider should strict `Role` and `CredentialsFile` config at the same place (since they both for authorization purpose) 👀 What do you think? 🤔

Reply:
`FunctionManifest` means the configuration of `kind: LambdaFunction` in the config-repo?
If so, I think it should keep in the PiepdConfig as is because this should be configured by a Piped operator.

Reply:
> FunctionManifest means the configuration of kind: LambdaFunction in the config-repo?

Yes, it is 👍